www/front_src/src/Resources/Graph/Performance/index.tsx

```diff
@@ -394,40 +427,56 @@ const PerformanceGraph = ({
             <Typography color="textPrimary" variant="body1">
               {title}
             </Typography>
-            <div className={classes.exportToPngButton}>
-              <ContentWithCircularLoading
-                alignCenter={false}
-                loading={exporting}
-                loadingIndicatorSize={16}
-              >
-                <>
+            <ButtonGroup className={classes.buttonGroup} size="small">
+              <Button className={classes.buttonLink}>
```

Comment:
You can use IconButton from centreon-frontend

Reply:
Ok, but the href isn't present on the IconButton ?

Reply:
In your function, you can use history.push from useHistory

Reply:
https://reactrouter.com/web/api/Hooks

Reply:
Ok thanks, I push a commit with your advices 👍